index.js

```diff
@@ -139,6 +141,16 @@ function displayCategories() {
   });
 }
 
+function filterTopicsByCategory() {
```

Comment:
The function name does not make much sense as it does not filter topics by category. 
But it can be changed in the following issue if you going to work on it 😸 

```suggestion
function activateCategoryBtns() {
```